core/src/main/java/org/testcontainers/containers/image/pull/policy/DefaultPullPolicy.java

```diff
@@ -0,0 +1,15 @@
+package org.testcontainers.containers.image.pull.policy;
+
+import java.util.Arrays;
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.image.ImageData;
+
+@Slf4j
+public class DefaultPullPolicy implements ImagePullPolicy {
```

Comment:
I can imagine that _this_ will be the place people go to look for javadocs on the default policy, so please could there be class javadocs describing it here? Maybe lift what's on `PullPolicy` to the classes, and reduce the method-level javadocs there to links.

Reply:
I have unresolved this because I don't think it is actually resolved.

@dmarkhas please keep the conversations open, the reporters will resolve them when they feel so :)

Reply:
@bsideup this class now contains javadocs that describe its function, why do you think it is not resolved?